ts/src/okx.ts

```diff
@@ -6251,6 +6251,29 @@ export default class okx extends Exchange {
         return response;
     }
 
+    async fetchPositionMode (symbol: Str = undefined, params = {}) {
+        /**
+         * @method
+         * @name okx#fetchPositionMode
+         * @description fetchs the position mode, hedged or one way, hedged for binance is set identically for all linear markets or all inverse markets
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} an object detailing whether the market is in hedged or one-way mode
+         */
+        const accounts = await this.fetchAccounts ();
+        const length = accounts.length;
+        if (length > 1) {
+            throw new ExchangeError (this.id + ' fetchPositionMode() can not detect position mode, because you have multiple accounts on this exchange. Please use fetchAccounts() to determine position mode');
```

Comment:
hum, this message is not very helpful, can't we require the account id in params or something when this happens? fetchAccounts does not return the same type so it might be tricky for the users

Reply:
fixed. now , only if there are several accounts, users get exception with instruction, and then they can `.fetchPositionMode (undefined, {accountId: '1234...'}`